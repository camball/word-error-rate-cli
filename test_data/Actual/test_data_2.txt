Agent: These is another tests.
Customer: Saw, thank you. My day have been grape.
Agent: How can I help your today?