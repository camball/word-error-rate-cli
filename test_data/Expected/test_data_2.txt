Agent: This is another test.
Customer: Ah, thank you. My day has been great.
Agent: Splendid. How can I help you today?